packages/react-renderer-demo/src/components/navigation/schemas/schema.js

```diff
@@ -57,6 +58,12 @@ const schema = [
     linkText: 'Optimization',
     link: 'optimization'
   },
+  {
+    title: 'Custom components demo',
```

Comment:
what about to remove `demo`?

Reply:
We already have custom components entry in the docs. I would like users to find the guide how to create the custom components before looking at this.

Reply:
hmm so maybe 'Custom components examples'?

Reply:
I think examples might be enough.